main.cpp

```diff
@@ -132,41 +134,45 @@ Gym::Member::~Member()
     std::cout << "Gym::Member Deconstructed \n";
 }
 
-double Gym::sellMonthlyPass(Gym::Member, double subscriptionPrice, int numMonths)
+double Gym::sellMonthlyPass(Gym::Member, double mSubscriptionPrice, int numMonths)
 {
     subscriptionPrice = mSubscriptionPrice;
     double totalPrice = subscriptionPrice * numMonths;
     return totalPrice;
 }
 
-void Gym::setPersonalTrainer(std::string personalTrainerName, float sessionLength, double sessionTime)
+void Gym::setPersonalTrainer(std::string mPersonalTrainerName, float sessionLength, double sessionTime)
 {
-    personalTrainerName = "Dave";
+    mPersonalTrainerName = "Dave";
     sessionLength = 1.5f;
     sessionTime = 2;
 
-    std::cout << "You have scheduled a session with: " << personalTrainerName << "\n" << "at: " << sessionTime << "\n" << "for: " << sessionLength << " hours\n";
+    std::cout << "You have scheduled a session with: " << mPersonalTrainerName << "\n" << "at: " << sessionTime << "\n" << "for: " << sessionLength << " hours\n";
 }
 
-double Gym::getUserFreq(double numDailyPeople, int numDays)
+double Gym::getUserFreq(double mNumDailyPeople, int numDays)
 {
-    double userFreq = numDailyPeople / numDays;
+    double userFreq = mNumDailyPeople / numDays;
     return userFreq;
 }
```

Comment:
Don't use `m` in the function parameter names
Skip the temporary local variable and just write `return numDailyPeople / numDays;` 
- [ ] revise